server/s3_bucket_metadata.cc

```diff
@@ -289,6 +293,32 @@ std::string S3BucketMetadata::get_tags_as_xml() {
   return tags_as_xml_str;
 }
 
+std::string S3BucketMetadata::get_bucket_versioning_status_as_xml() {
+  s3_log(S3_LOG_INFO, stripped_request_id, "%s Entry\n", __func__);
+  std::string versioning_status;
+  std::string versioning_status_as_xml_str;
+
+  if (bucket_versioning_status != "Unversioned") {
+    versioning_status +=
+        "<VersioningConfiguration "
+        "xmlns=\"http://s3.seagate.com/doc/2006-03-01/\">" +
```

Comment:
This should remain "s3.amazon.com".

Reply:
@sjain09 Is that the requirement? Do you see any issues with changing this to seagate?

Reply:
There is a comment from keith on this. See above comment, which explains it.

Reply:
Keith's comment -
>> It's correct as-is, we're implementing the AWS API and so we need to return the same responses. This indicates which version of the AWS S3 specification is being used. All S3-compatible servers use this same XML namespace.

Reply:
Also we need to be consistent with all other APIs too.

Reply:
Yup, reverting.